chart/apisix/templates/configmap.yaml

```diff
@@ -72,9 +73,14 @@ data:
       #    disk_path: "/tmp/disk_cache_two"
       #    cache_levels: "1:2"
 
-    #  allow_admin:                  # http://nginx.org/en/docs/http/ngx_http_access_module.html#allow
-    #    - 127.0.0.0/24              # If we don't set any IP list, then any IP access is allowed by default.
-    #    - 172.17.0.0/24
+      allow_admin:                  # http://nginx.org/en/docs/http/ngx_http_access_module.html#allow
```

Comment:
It is not a good design, we can not limit user config by predefined parameter.
Define a values parameter names `config`, then transform it's content to `config.yaml` would be better, it can give user full control to config.

Reply:
Okay, we can do this config.yaml configuration in another issue #7 .